commands/ThresholdCommand.java

```diff
@@ -0,0 +1,44 @@
+package org.usfirst.frc4904.standard.commands;
+
+
+import java.util.function.Supplier;
+import edu.wpi.first.wpilibj.command.Command;
+
+/**
+ *
+ */
+public class ThresholdCommand extends Command {
+	protected final Command command;
+	protected final Supplier<Double> axis;
```

Comment:
Now that this is a general command, this isn't necessarily an `axis` so we should refactor this to something else (maybe `supplier`).